elements/pfe-tabs/src/pfe-tabs.story.js

```diff
@@ -0,0 +1,150 @@
+import { storiesOf } from "@storybook/polymer";
+import * as storybookBridge from "@storybook/addon-knobs/polymer";
+import * as tools from "../../../.storybook/utils.js";
+
+const cleaner = require("clean-html");
+
+import PfeTabs from "../pfe-tabs";
+
+// import cpTheme from "../../../themes/cp-theme/cp-theme.js";
+
+const stories = storiesOf("Tabs", module);
+
+const defaultTab = tools.autoHeading(true);
+const defaultPanel =
+  tools.customTag({
+    tag: "h3",
+    content: tools.autoHeading()
+  }) +
+  tools.autoContent(5, 3) +
+  tools.component("pfe-cta", {}, [
+    {
+      content: "<a href='#'>Learn more</a>"
+    }
+  ]);
+
+// Define the template to be used
+const template = (data = {}) => {
+  return tools.component(PfeTabs.tag, data.prop, data.slots);
+};
+
+stories.addDecorator(storybookBridge.withKnobs);
+
+stories.add(PfeTabs.tag, () => {
+  let config = {};
+  // const props = PfeTabs.properties;
+  // Manually defining props but this can be done in a schema instead
+  const props = {
```

Comment:
@castastrophe - It looks like we don't have an orientation attribute on tabs. I think it's just a matter of adding a vertical attribute.

https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-tabs/pfe-tabs.js#L247

Reply:
There are styles based on a vertical boolean attribute so: `<pfe-tabs vertical>`

Reply:
Oh!  I see what you're saying; I just pushed up a fix for this 077a9477765be2d81e97e038dcfae2d5d3e7b77b